test/options/attributeSeparator/closing-bracket-position/closing-bracket-position.test.ts

```diff
@@ -0,0 +1,21 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { format } from 'prettier';
+import { plugin } from '../../../../src/index';
+
+describe('Options', () => {
+	describe('closingBracketPosition', () => {
```

Comment:
Please add two more tests (one for the default case and one for each of the two options) and rebuild the folder structure, so like the other two possible configurations